### PR TITLE
[FE] Improved UI Design: Navbar, Edit Item, Add Item

### PR DESCRIPTION
**Pull request by misterRenz**
## Task Link

- https://app.asana.com/0/1202718774787420/1202822440884353/f

## Commands

- npm start

## Pre-conditions

- The Frontend developer must be in the fe folder.

## Expected Output

- It will show a new design for Navbar design, Add Item Page, Edit Item Page, Add User, and Edit User

## Notes

- None

## Screenshots
image
image
image
image
image

### CODE REVIEW

File: frontend/src/components/navbar/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { useNavigate } from "react-router-dom";
-
+import "./index.css";
 const NavBar = () => {
```
Comment (ziontavera) [inline frontend/src/components/navbar/index.js]:
```suggestion
import "./index.css";

const NavBar = () => {
```

File: frontend/src/pages/register/index.js [hunk 2]
```diff
@@ -13,14 +13,13 @@ const RegisterPage = () => {
...
-            <h2>Add User</h2>
-            <form className="ui form">
+            <form className="form-add-user">
+              <h1 className="page-title">Add User</h1>
               <div className="field">
                 <label>Name</label>
                 <input
                   type="text"
                   placeholder="Enter Name"
-                  name="name"
```
Comment (ziontavera) [inline frontend/src/pages/register/index.js:23]: is this not needed?

Comment (misterRenz) [reply frontend/src/pages/register/index.js:23]: Yes

File: frontend/src/modals/add-user/index.js [hunk 3]
```diff
@@ -25,12 +25,17 @@ const AddUserModal = ({ addUser }) => {
       return;
     }
 
-    alert(`Successfully added ${addUser}`)
+    if (addUser.trim() === "") {
```
Comment (ziontavera) [inline frontend/src/modals/add-user/index.js]:
```suggestion
    if (!addUser.trim()) {
```

**Review by ziontavera — APPROVED** [review]

**Pull request merged**